src/experimental/tools/mediaCapabilitiesProber/api/probeMediaConfiguration.ts

```diff
@@ -71,9 +69,19 @@ async function probeMediaConfiguration(
     const probeWithBrowser = probers[browserAPI][0];
     const wantedLogLevel = probers[browserAPI][1];
     if (probeWithBrowser) {
-      promises.push(probeWithBrowser(config).then(([statusNumber, result]) => {
+      promises.push(probeWithBrowser(config).then(([currentStatus, result]) => {
         resultsFromAPIS.push({ APIName: browserAPI, result });
-        globalStatusNumber = Math.min(globalStatusNumber, statusNumber);
+
+        // /!\ This code might be too smart. But it should work.
+        if (
+          globalStatus !== ProberStatus.NotSupported && (
+            currentStatus === ProberStatus.NotSupported ||
+            currentStatus === ProberStatus.Unknown ||
+            globalStatus == null
```

Comment:
I may have been too smart there (I didn't think of a possible hole there though, but maybe there is).

Probably should be better written

Reply:
The ultra-readable version of it. Maybe too heavy ...
``
        switch (currentStatus) {
          case ProberStatus.NotSupported:
            globalStatus = ProberStatus.NotSupported;
            break;
          case ProberStatus.Unknown:
            if (
              globalStatus == null ||
              globalStatus === ProberStatus.Supported
            ) {
              globalStatus = ProberStatus.Unknown;
            }
            break;
          case ProberStatus.Supported:
            if (globalStatus == null) {
              globalStatus = ProberStatus.Supported;
            }
            break;
          default:
            if (globalStatus == null) {
              globalStatus = ProberStatus.Unknown;
            }
        }
``

Reply:
`        switch (currentStatus) {
          case ProberStatus.NotSupported:
            globalStatus = ProberStatus.NotSupported;
            break;
          case ProberStatus.Unknown:
            if (
              globalStatus == null ||
              globalStatus === ProberStatus.Supported
            ) {
              globalStatus = ProberStatus.Unknown;
            }
            break;
          case ProberStatus.Supported:
            if (globalStatus == null) {
              globalStatus = ProberStatus.Supported;
            }
            break;
          default:
            if (globalStatus == null) {
              globalStatus = ProberStatus.Unknown;
            }
        }`

Reply:
The heavier-readable version of it ... :
``switch (currentStatus) {
          case ProberStatus.NotSupported:
            globalStatus = ProberStatus.NotSupported;
            break;
          case ProberStatus.Unknown:
            if (
              globalStatus == null ||
              globalStatus === ProberStatus.Supported
            ) {
              globalStatus = ProberStatus.Unknown;
            }
            break;
          case ProberStatus.Supported:
            if (globalStatus == null) {
              globalStatus = ProberStatus.Supported;
            }
            break;
          default:
            if (globalStatus == null) {
              globalStatus = ProberStatus.Unknown;
            }
        }``